pkg/cmd/join/exec.go

```diff
@@ -355,7 +355,8 @@ func (o *Options) run() error {
 		}
 	}
 
-	fmt.Fprintf(o.Streams.Out, "Please log onto the hub cluster and run the following command:\n\n"+
+	fmt.Fprintf(o.Streams.Out, "\nYour managed cluster %s has joined the Hub.\n", o.values.ClusterName)
```

Comment:
since  ManagedClusterAutoApproval  is disable by default, we might say
Please log onto the hub cluster and run the following command: at first and then say it is not needed when If the ManagedClusterAutoApproval feature is not enabled.

Reply:
updated